mapstruct-spi-accessor-naming/usage/pom.xml

```diff
@@ -21,42 +21,52 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>src</artifactId>
-    <version>1.0.0</version>
+
     <parent>
-        <groupId>org.mapstruct.examples.spi.accessornaming</groupId>
-        <artifactId>parent</artifactId>
+        <groupId>org.mapstruct.examples.spi</groupId>
+        <artifactId>accessornaming-parent</artifactId>
```

Comment:
In Eclipse I can't import two modules with the same artifactId, so I usually prefix them there... :-/